frontend/src/reducer/notificationPage.js

```diff
@@ -6,20 +6,23 @@ import {
   NOTIFICATION,
   NOTIFICATION_LIST,
   SET,
-  UPDATE
+  READ,
+  NOTIFICATION_NOT_READ_COUNT
 } from '../action-creator.sync.js'
 
 const defaultNotificationsObject = {
   list: [],
   hasNextPage: false,
-  nextPageToken: ''
+  nextPageToken: '',
+  notificationNotReadCount: 0
 }
 
 // FIXME - GB - 2020-07-30 - We can't use the global serializer in this case because it doesn't handle nested object
 // See https://github.com/tracim/tracim/issues/3229
 export const serializeNotification = no => {
```

Comment:
Later in the code, the name `notification` is used, I would suggest to rename `no` to `notification`

Reply:
naming a var "no" will *for sure* result in a misunderstanding. Short names must be avoided.